controllers/rbac/aggregation_test.go

```diff
@@ -139,14 +140,8 @@ var _ = Describe("Aggregation Tests", func() {
 						Verbs:     []string{"all"},
 					},
 				}
-				oldRole.OwnerReferences[0].UID = "1234"
```

Comment:
why this part is not necessary anymore?

Reply:
good catch, I don't know why I removed it, added it again 👍🏼